texastribune/data-visuals-create
# Remove debounce

### Changed
- `graphic/app/scripts/embeds/frames.js` - don't pass debounced function to resize listener, simplify code #29

### To test
Test it the way you tested the initial pym -> frames release! 

Create static ai2html, JS scripted, and straight up HTML graphics, deploy, and embed them in a test article in the CMS. 

Resize the window and switch to mobile to see if the iframe resizes OK (you should see a continuous resize.)
